### ISSUE TITLE
[declarations] fare double check del file missing_accessibility_urls.csv dopo primo processing

### ISSUE THREAD

**Issue by dennisangemi** [opendatasicilia/accessibility-monitoring#5]
ho notato che `missing_accessibility_urls.csv` contiene dei falsi positivi e lanciare l'intero script prendendo questo file come input riduce il numero di PA per cui non sono stati trovati riferimenti all'accessibilità. Integrare questo processo di double check nello script principale

**Comment by dennisangemi** [comment]
fatto con https://github.com/opendatasicilia/accessibility-monitoring/commit/133d37fcf98fe6de8dc458bb4d3b3f649859ff0f ma necessita di test prima di chiudere
